### PR TITLE
improved error message

### PR DESCRIPTION
**Pull request by yaakovi**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/22187

## Description
test passed without any changes.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

File: TestPlaybooks/playbook-EmailReputationTest.yml [hunk 1]
```diff
@@ -277,7 +277,7 @@ tasks:
       brand: ""
     scriptarguments:
       message:
-        simple: Didn't get proper score for indicator
+        simple: 'Didn''t get proper score for indicator. Expected: 2, Got: ${res_score.DBotScore.[0].Score}'
```
Comment (ShahafBenYakir) [inline TestPlaybooks/playbook-EmailReputationTest.yml]:
```suggestion
        simple: Didn't get proper score for indicator. Expected: 2, Got: ${res_score.DBotScore.[0].Score}'
```

Comment (amshamah419) [reply TestPlaybooks/playbook-EmailReputationTest.yml]: Nice

Comment (yaakovi) [reply TestPlaybooks/playbook-EmailReputationTest.yml]: this is how the playbook were downloaded from demisto

Comment (ShahafBenYakir) [reply TestPlaybooks/playbook-EmailReputationTest.yml]: 🤷‍♂

**Pull request merged**